packages/calcite-components/src/custom-theme/card.tsx

```diff
@@ -0,0 +1,24 @@
+import { placeholderImage } from "../../.storybook/placeholderImage";
+import { html } from "../../support/formatting";
+
+export const tokens = {
```

Comment:
nitpick: maybe this should just be named `export const tokens` so it doesn't need to be renamed during the import.

If we are planning to do this in other components it would be good to set a standard here.